manifests/init.pp

```diff
@@ -65,9 +65,11 @@
 #
 # @param dns_stub_listener
 #   Takes a boolean argument or one of "udp" and "tcp".
+#   Setting it to `'absent'` will remove `DNSStubListener` existing entries from the configuration file-
 #
 # @param dns_stub_listener_extra
 #   Additional addresses for the DNS stub listener to listen on
+#   Setting it to `['absent']` will remove `DNSStubListenerExtra` existing entries from the configuration file-
```

Comment:
The `[]` look a bit odd.

Reply:
Maybe `Variant[Array[String[1],Enum['absent']]` ?

Reply:
well the param is Array only at the moment. Should I add `Enum['variant']` to the variants?